extensions/wikia/AdEngine/js/run/mercury.run.js

```diff
@@ -48,7 +48,10 @@ require([
 			rubiconFastlane.call();
 		}
 
-		if (geo.isProperGeo(instantGlobals.wgAdDriverOpenXBidderCountries)) {
+		if (
```

Comment:
nitpick: what do you think about adding a TODO with removal of this double check after decision whether we use openX in prebid or not will be made?